style.css

```diff
@@ -51,6 +51,60 @@ a {
   display: none;
 }
 
+.close-button {
+  background-color: transparent;
+  border: none;
+  color: #fff;
+  cursor: pointer;
+  font-size: 18px;
+  padding: 0;
+}
+
+.close-button:hover {
+  color: #8c8a8a;
+}
+
+@media (max-width: 768px) {
+  .nav-list {
+    display: none;
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100vh;
+    background-color: #6070ff;
+    mix-blend-mode: multiply;
+    backdrop-filter: blur(8px);
+    list-style: none;
+    padding: 0;
+    margin: 0;
+    z-index: 999;
+  }
```

Comment:
- [ ] The mobile menu is expected to be fixed and not scrolled. Currently, when I scroll on mobile view, the menu scrolls along. It's expected to stay fixed and only closed when an item in the list or the close(X) button is clicked. Could you work on making the mobile menu fixed and not scrollable? 👍🏽
- [ ] Also, please align the mobile menu navigation list items to the left just as it is in the original design. 👍🏽


**Your Design**

<img width="468" alt="Screenshot 2023-05-17 at 8 17 40 AM" src="https://github.com/Eccb7/Portfolio2.0/assets/50941074/ef52cff1-f284-469c-b15b-5d0f8e88e34d">

**The Original Design**

<img width="373" alt="Screenshot 2023-05-17 at 8 30 04 AM" src="https://github.com/Eccb7/Portfolio2.0/assets/50941074/c8448b2d-2510-41f4-814a-a147781b821a">

**Note: The list items are aligned to the left and not center**